experiments/cql_compare.py

```diff
@@ -15,6 +15,7 @@
 
 from replay.data_preparator import DataPreparator, Indexer
 from replay.experiment import Experiment
+from replay.mdp_dataset_builder import MdpDatasetBuilder
```

Comment:
What about placing `MdpDatasetBuilder` in cql.py (before CQL class for example)? Code in `replay` directory  typically model-independent. In case of positive response keep in mind moving `tests/test_mdp_dataset_builder.py` to `tests/models/test_cql.py`